[2022-06-03 16:51:47,180] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: nyc_bus_breakdown_pipeline.processing_data manual__2022-06-03T16:51:38.416391+00:00 [queued]>
[2022-06-03 16:51:47,189] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: nyc_bus_breakdown_pipeline.processing_data manual__2022-06-03T16:51:38.416391+00:00 [queued]>
[2022-06-03 16:51:47,190] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-03 16:51:47,190] {taskinstance.py:1358} INFO - Starting attempt 1 of 2
[2022-06-03 16:51:47,191] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-03 16:51:47,205] {taskinstance.py:1378} INFO - Executing <Task(PythonOperator): processing_data> on 2022-06-03 16:51:38.416391+00:00
[2022-06-03 16:51:47,210] {standard_task_runner.py:52} INFO - Started process 1668 to run task
[2022-06-03 16:51:47,214] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'nyc_bus_breakdown_pipeline', 'processing_data', 'manual__2022-06-03T16:51:38.416391+00:00', '--job-id', '140', '--raw', '--subdir', 'DAGS_FOLDER/nycbus_etl.py', '--cfg-path', '/tmp/tmp_8fziqc3', '--error-file', '/tmp/tmp5pqa5ipb']
[2022-06-03 16:51:47,215] {standard_task_runner.py:80} INFO - Job 140: Subtask processing_data
[2022-06-03 16:51:47,274] {task_command.py:370} INFO - Running <TaskInstance: nyc_bus_breakdown_pipeline.processing_data manual__2022-06-03T16:51:38.416391+00:00 [running]> on host 6cc5764fb927
[2022-06-03 16:51:47,554] {taskinstance.py:1572} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=nyc_bus_breakdown_pipeline
AIRFLOW_CTX_TASK_ID=processing_data
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T16:51:38.416391+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-03T16:51:38.416391+00:00
[2022-06-03 16:51:48,731] {base.py:68} INFO - Using connection ID 's3_conn' for task execution.
[2022-06-03 16:51:48,732] {base_aws.py:210} INFO - Credentials retrieved from extra_config
[2022-06-03 16:51:50,632] {python.py:173} INFO - Done. Returned value was: None
[2022-06-03 16:51:50,646] {taskinstance.py:1401} INFO - Marking task as SUCCESS. dag_id=nyc_bus_breakdown_pipeline, task_id=processing_data, execution_date=20220603T165138, start_date=20220603T165147, end_date=20220603T165150
[2022-06-03 16:51:50,681] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-03 16:51:50,724] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
